[docs] Document the release management principles (#9974)
organicmaps/organicmaps: docs/RELEASE_MANAGEMENT.md

### Schedule vs Scope

Each release aims to include all features that have been merged into the master branch before the release cut-off date and have not caused regressions during the QA validation process. If certain changes are not included in the current release, they will be postponed to the next one.

The general recommendation is to merge large changes immediately after the release to allow the entire month for fixing potential regressions and stabilizing the system.

### Release Manager

Each release has one person appointed as the Release Manager to oversee the execution of the release management process.

It is the duy of Release Manager to drive the enture process to the successful completion, utilizing all available resources and means. Specific tasks may be delegated to individual team members as required.

The person holds the authority to make final decisions on cut-off and release dates, and can exclude or revert changes to ensure the release schedule is met.

> ### Collaboration
> 
> Volunteer Contributors and Project Managers are encouraged to plan accordingly and communicate clearly with the Release Manager about their schedules, as well as any potential risks or regressions after merging each feature.
> 
> ### Scoping
> 
> Each release should have a dedicated 20YY.MM milestone created on GitHub. Contributors and Project Managers are encouraged to use it for organizing their work.
> 
> ### Blockers
> 
> Any regressions discovered must be filed as blocking bug tickets and added to the milestone with a clear and understandable description. Contributors are encouraged to proactively revert their changes if it is unrealistic to stabilize them before the cut-off date.
> 
> ### Tracking Ticket
> 
> Each release progress is tracked using the Release Tracking Issue ticket on GitHub, which contains a checklist of items to be executed in the correct order.
> 
> ### Communication
> 
> Any decision made must be documented in writing within the ticket, ensuring that the motivation and reasoning are clear and understandable to all.

Reviewer: ```suggestion
### Release Tracking

Each release should have its own Release Tracking Issue on GitHub, which contains a schedule and a checklist of items to be executed in the correct order.

Also each release should have a dedicated 20YY.MM github milestone. All contributors are encouraged to use it for organizing their work and to communicate clearly with the Release Manager about their schedules, as well as any potential risks or regressions after merging each feature.

Any regressions discovered must be filed as blocking bug tickets and added to the milestone with a clear and understandable description. Contributors are encouraged to proactively revert their changes if it is unrealistic to stabilize them before the cut-off date.

Any decision made must be documented in writing within the Release Tracking Issue, ensuring that the motivation and reasoning are clear and understandable to all.
```

IMHO it was too many sub-titles each containing just one line of text.

Reviewer: Maybe the "Blockers" is the only item deserving its own heading..
